Resources/Maps/_NF/Shuttles/Nfsd/wendigo.yml

```diff
@@ -768,6 +768,20 @@ entities:
     - type: Transform
       pos: 9.5,4.5
       parent: 1
+- proto: BaseAdvancedPen
+  entities:
+  - uid: 471
+    components:
+    - type: Transform
+      rot: -28.27433388230815 rad
+      pos: -1.3030288,-3.779652
+      parent: 1
+  - uid: 472
+    components:
+    - type: Transform
+      rot: -21.991148575128555 rad
+      pos: 2.3636386,-3.758803
+      parent: 1
```

Comment:
I am enormously intrigued by the presence of these two pens. Contraband pens on an NFSD ship?! How did this happen? 😄

Reply:
Contraband pens woops

Reply:
what the hell is an advanced pen?

Reply:
It's the base prototype for e-daggers, explosive pens and hypopens. It should probably be abstract. Blame upstream.